yb-voyager/src/srcdb/pg_dump_extract_schema.go

```diff
@@ -135,12 +135,14 @@ func parseSchemaFile(exportDir string, schemaDir string, exportObjectTypesList [
 				objSqlStmts["SEQUENCE"].WriteString(stmts)
 			case "INDEX", "INDEX ATTACH":
 				objSqlStmts["INDEX"].WriteString(stmts)
-			case "TABLE", "DEFAULT", "CONSTRAINT", "FK CONSTRAINT", "CHECK CONSTRAINT":
+			case "TABLE", "DEFAULT", "CONSTRAINT", "FK CONSTRAINT", "CHECK CONSTRAINT", "ROW SECURITY":
```

Comment:
What are we ultimately doing with ROW SECURITY? Do we want to categorize it as unsupported or something?

Reply:
We are categorizing it as a part TABLE object as it is an ALTER TABLE but it is supported in YB so no need to report it as unsupported or anything.

Reply:
Oh okay, so before it was going to uncategorized?

Reply:
right